src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -221,7 +259,27 @@ export default {
 				this.$kvTrackEvent('Register', 'error-register-social-form-invalid-input');
 			}
 		},
+		fetchStrategicPartnerLoginInfoByPageId() {
+			this.apollo.query({
+				query: StrategicPartnerLoginInfoByPageId,
+				variables: { pageId: this.partnerContentId }
+			}).then(response => {
+				const { logo } = response.data.strategicPartnerLoginInfoByPageId;
+				this.strategicPartnerLogo = {
+					url: logo.url,
+					alt: logo.alt
+				};
+			}).catch(error => {
+				console.error('Error fetching strategic partner info:', error);
+			});
+		}
 	}
 
 };
 </script>
+
+<style scoped>
+.logo {
+	box-shadow: 0 0 18px rgba(0, 0, 0, 0.2);
```

Comment:
Can we use our pre-defined shadows in TW?

Reply:
None of them look right for this design.  They don't provide adequate contrast against the black and make the logos look washed out.

Reply:
Sara gave the shadow a deeper color for this case in figma than anything supported by tailwind.